q2_demux/plugin_setup.py

```diff
@@ -0,0 +1,44 @@
+import importlib
+
+import qiime
+import qiime.plugin
+from q2_types.sample_data import SampleData
+from q2_types.per_sample_sequences import SequencesWithQuality
+
+import q2_demux
+from ._type import RawSequences
+from ._format import EMPMultiplexedDirFmt
+
+
+plugin = qiime.plugin.Plugin(
+    name='demux',
+    version=q2_demux.__version__,
+    website='https://github.com/qiime2/q2-demux',
+    package='q2_demux',
+    user_support_text=None,
+    citation_text=None
+)
+
+plugin.register_semantic_type(RawSequences)
+
+plugin.register_semantic_type_to_format(
+    RawSequences,
+    artifact_format=EMPMultiplexedDirFmt
+)
+
+plugin.methods.register_function(
+    function=q2_demux.emp,
+    inputs={'seqs': RawSequences},
+    parameters={'barcodes': qiime.plugin.MetadataCategory,
+                'rev_comp_barcodes': qiime.plugin.Bool,
+                'rev_comp_mapping_barcodes': qiime.plugin.Bool},
+    outputs={'per_sample_sequences': SampleData[SequencesWithQuality]},
+    name='Demultiplex sequence data generated with the EMP protocol.',
+    description=('Demultiplex sequence data (i.e., map barcode reads to '
+                 'sample ids) for data generated with the Earth Microbiome '
+                 'Project amplicon sequencing protocol. Details about this '
```

Comment:
add "(EMP)" (no quotes) after "Earth Microbiome Project" so the acronym is defined somewhere